packages/salesforcedx-vscode-apex/src/oas/generationStrategy/generationStrategy.ts

```diff
@@ -61,6 +62,9 @@ export abstract class GenerationStrategy {
     this.beta = betaInfo;
   }
 
+  async initialize(): Promise<void> {
```

Comment:
we need the eslint rule for this.  Let's put access modifier keywords on all methods.

https://typescript-eslint.io/rules/explicit-member-accessibility/

LMK if you don't want to add it on this PR and I'll do it separately.